src/App.tsx

```diff
@@ -1,8 +1,99 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
 import './App.css';
+import { Todolist } from './Todolist/Todolist';
+import { getTodos, getUsers } from './api/api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setTodos] = useState<TodoWithUser[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<TodoWithUser[]>([...todos]);
+  const [isLoading, setLoading] = useState(false);
+
+
+  const isloaded = async () => {
+    setLoading(true);
+
+    const [todosFromServer, users] = await Promise.all(
+      [getTodos(), getUsers()],
+    );
+
+    const preparedTodos = todosFromServer.map((todo: Todo) => {
```

Comment:
если описать что отдает функция getTodos, то думаю тут уже не нужно
```suggestion
    const preparedTodos = todosFromServer.map((todo) => {
```